grails-app/services/org/pih/warehouse/core/LocalizationService.groovy

```diff
@@ -69,51 +69,11 @@ class LocalizationService {
      */
     Properties getMessagesProperties(Locale locale) {
         Properties messagesProperties = new Properties()
-        def messagesPropertiesFilename = (locale && locale.language != "en" && locale.language != 'null') ? "messages_${locale.language}.properties" : "messages.properties"
+        def messagesPropertiesFilename = (locale && locale.language != "en" && locale.language != 'null') ? "messages_${locale.toString()}.properties" : "messages.properties"
```

Comment:
in what situations do we want `locale.toString()` vs locale.getDisplayName()`? I see we use both

Reply:
We want to find a message properties file corresponding to the current locale. `toString()` concatenates language code with country code together with an underscore (or returns just language code if no country code). So for Spanish, we are getting just `es` -> `messages_es.properties` and for Mexican Spanish, we will get `es_MX` -> `messages_es_MX.properties`